composer.json

```
@@ -9,9 +9,9 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
         "codeception/codeception": "^5.0",
-        "codeception/lib-innerbrowser": "^3.0",
+        "codeception/lib-innerbrowser": "^4.0",
```

With this change we can fully support PHP 8.0 without breaking change.

```suggestion
        "codeception/lib-innerbrowser": "^3.0 || ^4.0",
```

---

OK, I've made requested changes with changelog update (added [Unreleased] into 4.0 changelog, if it's ok)
So check it out and give me some info if some more changes are required. Thx.